stdlib/public/runtime/Win32.cpp

```diff
@@ -71,4 +76,61 @@ _swift_win32_copyWideFromUTF8(const char *str) {
   return nullptr;
 }
 
+#pragma mark - DbgHelp library thread-safety
+
+static LazyMutex dbgHelpMutex;
+static HANDLE dbgHelpHandle = nullptr;
+
+void _swift_win32_withDbgHelpLibrary(
+  void (* body)(HANDLE hProcess, void *context), void *context) {
```

Comment:
Does it make any sense to move the `hDebugHelp` into this function as a `static` and use `InitOnceExecuteOne` (or rather `std::once`) to initialize it?

Reply:
In the Swift runtime we should be using `swift::once` from `swift/Threading/Once.h`.

Reply:
I intentionally don't use `swift::once()` here because failures may be transient.